dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/AuthorityEntryLinkRepository.java

```diff
@@ -54,6 +55,7 @@ public HALResource wrapResource(AuthorityEntryRest model, String... rels) {
         return new AuthorityEntryResource(model);
     }
 
+    @PreAuthorize("hasAuthority('EPERSON')")
```

Comment:
`hasAuthority('EPERSON')` is awkwardly named, in my opinion.  EPerson is a type of object in DSpace and not a permission level.  It's confusing to see, for example, `hasPermission(#id, 'EPERSON', 'READ')` and `hasAuthority('EPERSON')` alongside one another, as the word EPERSON has two different meanings (the first is a specific type of object, the second seems to be implying you are "authenticated" and don't necessarily have specific rights on an EPERSON object).

Should we consider renaming this to `hasAuthority('AUTHENTICATED')`?  If I understand this, that's the true meaning -- you simply need to be authenticated to access this endpoint.